src/App.java

```diff
@@ -1,31 +1,262 @@
+              the color text for
+            - message (type String): the message to embed in formatted color codes
+
+        Returns:
+            - (type String): the colored text
+    */
+    public String returnColoredText(String colorName, String message) {
+        return getColorCodes(colorName) + message + ANSI_RESET;
+    }
+
+    /*
+        This is a function to print the instructions for the game
+
+        Parameters:
+            - none
+
+        Returns:
+            - none
+    */
     public void printInstructions() {
-        
+        // ASCII art from https://patorjk.com/software/taag/#p=display&f=RubiFont&t=Madlibs
+        printColoredLn("green", """
+╭──────────────────────────────────────╮
+│                                      │
+│ ▗▖  ▗▖ ▗▄▖ ▗▄▄▄ ▗▖   ▗▄▄▄▖▗▄▄▖  ▗▄▄▖ │
+│ ▐▛▚▞▜▌▐▌ ▐▌▐▌  █▐▌     █  ▐▌ ▐▌▐▌    │
+│ ▐▌  ▐▌▐▛▀▜▌▐▌  █▐▌     █  ▐▛▀▚▖ ▝▀▚▖ │
+│ ▐▌  ▐▌▐▌ ▐▌▐▙▄▄▀▐▙▄▄▖▗▄█▄▖▐▙▄▞▘▗▄▄▞▘ │
+│                                      │
+╰──────────────────────────────────────╯""");
+
+        // Unicode symbols for formatting looks like https://www.w3.org/TR/xml-entity-names/025.html
+        printColoredLn("green", "╭────────────────╮");
+        printColoredLn("green", "│  Instructions  │");
+        printColoredLn("green", "├────────────────╯"); //  ╰
+
+        // bullet point icon from  https://www.merriam-webster.com/grammar/how-to-use-bullet-points
+        System.out.println(returnColoredText("green", "│") + " • Enter your name");
+        System.out.println(returnColoredText("green", "│") + " • Enter the prompted words to create a funny story!");
+        System.out.println(returnColoredText("green", "│") + " • Repeat until you're tired of laughing");
+    }
+
+    /*
+        This is a function to print the greeting for the game
+
+        Parameters:
+            - none
+
+        Returns:
+            - none
+    */
+    public void printGreeting() {
+        printColoredLn("green","├──────────────────────────╮"); // ╭
+        printColoredLn("green","│    Welcome to Madlibs    │");
+        printColoredLn("green","├──────────────────────────╯"); //  ╰
+        System.out.println(returnColoredText("green", "│") + " What is your name?");
+
+        String userName = getUserInput();
+
+        printColoredLn("green","├────────────────────");
+        System.out.println(returnColoredText("green", "│") + " Hello, " + returnColoredText("green", userName) + "!");
+        System.out.println(returnColoredText("green", "│") + " Let's play a game!");
+    }
+
+    /*
+        This is a function to run and play the madlib game
+
+        Parameters:
+            - none
+
+        Returns:
+            - none
+    */
+    public void doMadlib() {
+        // I'll create an array of the terms I need, and loop
+        // through it to create a map of terms for the user's input
+
+        // https://www.w3schools.com/java/java_arrays.asp
+        String[] terms = {
```

Comment:
Would this extend to stories where you need to ask for a noun twice, or would you need some sort of internal way to distinguish them?